app/views/partials/decommissioning_alert_banner.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.connector.RepoType
+
+@(repoType: RepoType)
+
+<div class="alert alert-warning">
+ <strong>
+  This @{repoType.asString.toLowerCase} is marked for decommissioning, see <a id="decommission-state" href="/service/catalogue-frontend/commissioning-state">commissioning state</a> for progress. If this is unexpected contact <a href="https://hmrcdigital.slack.com/messages/team-platops" target=\"_blank\" rel=\"noreferrer noopener\">#team-platops</a> for help.
```

Comment:
Should we just hardcode the message to "Service" since this is the only type supported in Decommissioning? Otherwise it could say "This other is marked for decommissioning".

Reply:
There's an icon we usually show on links that open a new tab

Reply:
Done